velox/dwio/common/InputStream.cpp

```diff
@@ -72,7 +72,7 @@ void ReadFileInputStream::read(
   }
   logRead(offset, length, purpose);
   auto readStartMicros = getCurrentTimeMicro();
-  std::string_view data_read = readFile_->pread(offset, length, buf);
+  std::string_view data_read = readFile_->read(offset, length, buf);
```

Comment:
We cannot change this to use something thread unsafe

Reply:
revert